business/model/fsi_details.go

```diff
@@ -0,0 +1,25 @@
+package model
+
+type FsiStruct struct {
+	Plans      []FsiDetailPlans       `json:"plans"`
```

Comment:
Use ur own model for plans as per the expected response

Reply:
done